package co.yapp.orbit.fortune.application.exception;

public class FortuneCreateInvalidUserException extends RuntimeException {
    public FortuneCreateInvalidUserException(String message) {
        super(message);
    }
}
